\clearpage
\pagenumbering{arabic}
\setcounter{page}{1}

\section*{Supplementary Tables}

\setcounter{table}{0}
\renewcommand{\thetable}{S\arabic{table}}


%\begin{longtable}[c]{|p{3cm}|p{1.5cm}|p{1.0cm}|p{1.0cm}|p{1.0cm}|p{1.0cm}|p{1.0cm}|p{1.0cm}|p{1.0cm}|p{1.0cm}|}
\begin{table}[h]
\centering
\caption{\bf{Predictive comparison of \CorShrink{}, \Robocov{} and sample correlation estimators for a GTEx gene. We report Mean Absolute Deviation (MAD) and Root Mean Squared Deviation (RMSD) metrics between an estimator (e.g., sample correlation matrix, \CorShrink{} and \Robocov{}) computed on the gene expression data (GTEx project) for half of the individuals (training set) and the sample correlation matrix computed from other half (testing set) of all individuals. Results are averaged over 30 such different training/testing data-splits with the standard errors reported in brackets.}}
\label{tab:gtex_predictive} 
\begin{tabular}{|c|c|c|} \hline 
Method & MAD & RMSE  \\ \hline 
Sample-Est & 0.30 (0.01)  & 0.47 (0.02)  \\ \hline 
CorShrink & 0.24 (0.01) & 0.35 (0.01)  \\ \hline 
Robocov  & 0.25 (0.01)  & 0.36 (0.01)  \\ 
\hline 
\end{tabular}
\end{table}


\newpage
\begin{table}[h]
\centering
\caption{\bf{Pathway enrichment analysis of  \Robospan{}, \pRobospan{} and \Corspan{} genes. Pathway enrichment is performed using the ConsensusPathDB database\cite{Kamburov2012}. Only the top $5$ non-redundant and statistically significant (q-value $<$ 0.05) pathways for a gene set are reported.}}
\label{tab:pathway} 
\begin{tabular}[c]{|p{2cm}|p{4cm}|}
\toprule
 Gene Set & Top pathways\\ \hline
\multirow{1}{16em}{\Robospan{}}  & Interferon signaling (1.1e-18), Immune system (3.1e-08), HSF1 activation (1.1e-07), Antigen processing and presentation (2.4e-07), Allograft rejection (1.5e-06) \\ \hline

\multirow{1}{16em}{\pRobospan{}}  & Immune system (2.7e-21), Interferon signaling (3.4e-15), Innate immune system (5.1e-12),  TNF signaling pathway (7.9e-11), Neutrophil degranulation (2.0e-10) \\ \hline 

\multirow{1}{16em}{\Corspan{}}  & Interferon signaling  (1.1e-17), Immune system (1.6e-07), Antigen processing and presentation (1.2e-05), HSF1 activation (4.1e-05), Neutrophil degranulation (1e-04),  \\ \hline 
\end{tabular}
\end{table}

\newpage
\begin{table}[h]
\centering
\caption{\bf{List of $11$ blood and autoimmune traits (5 blood traits and 6 autoimmune traits) analyzed in this paper.}}
\label{tab:list_blood_traits}
\begin{tabular}[c]{|p{2cm}|p{4cm}|}
\toprule
  Annotation & Traits\\ \hline
Blood traits & Red blood Cell Distribution Width (UKBB\cite{Bycroft2018}), Red blood Cell Count (UKBB\cite{Bycroft2018}), White blood Cell Count (UKBB\cite{Bycroft2018}), Platelet Count (UKBB\cite{Bycroft2018}), Eosinophil Count (UKBB\cite{Bycroft2018}) \\ \hline
Immune traits & Ulcerative Colitis\cite{jostins2012}, Rheumatoid Arthritis\cite{okada2014}, Celiac\cite{Dubois2010}, Lupus\cite{Bentham2015}, Crohn's disease\cite{jostins2012}, Auto Immune and Inflammatory traits  \\ \hline 
\end{tabular}
\end{table}


\newpage
\begin{table}[h]
\centering
\caption{\bf{S-LDSC results for SNP annotations corresponding to \Robospan{}, \pRobospan{}, \Corspan{} and SEG-Blood gene sets for blood and autoimmune traits. Standardized Effect sizes ($\tau^{\star}$) and Enrichment (E) of $8$ SNP annotations corresponding to 4 gene sets (\Robospan{}, \pRobospan{}, \Corspan{} and SEG-Blood\cite{Finucane2018}) and 2 SNP annotations corresponding to 5kb and 100kb window based SNP-to-gene linking strategies for each gene set.Results for all annotations are conditional on $86$ baselineLD-v2.1 annotations. Reports are meta-analyzed across $11$ Blood and Autoimmune traits.}}
\label{tab:Robocov_marginal} 
\begin{tabular}[c]{|p{2cm}|p{1cm}|p{0.8cm}|p{1.3cm}|p{0.5cm}|p{1cm}|p{1.3cm}|}
 \multicolumn{7}{c}{\Robospan{}}  \\ \hline
& $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.6\%)$} & 0.086 & 0.024 & 0.00048 & 2.7 & 0.16 & 1.5e-07 \\
\multirow{1}{16em}{100kb $(10\%)$} & 0.12 & 0.03 & 7.9e-05 & 2.3 & 0.12 & 2e-09 \\
\hline 
 \multicolumn{7}{c}{\pRobospan{}}  \\ \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.3\%)$} & 0.096 & 0.028 & 0.00057 & 3.2 & 0.22 & 9.3e-08 \\
\multirow{1}{16em}{100kb $(9.9\%)$} & 0.11 & 0.034 & 0.0011 & 2.4 & 0.12 & 5.5e-09 \\
\hline 
\multicolumn{7}{c}{\Corspan{}}  \\ \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.5\%)$} & 0.04 & 0.024 & 0.093 & 2.4 & 0.15 & 4.8e-07 \\
\multirow{1}{16em}{100kb $(9.8\%)$} & 0.038 & 0.02 & 0.059 & 2.1 & 0.1 & 1.7e-08 \\
 \hline 
\multicolumn{7}{c}{SEG-Blood}  \\ \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.7\%)$} & 0.24 & 0.036 & 7.6e-11 & 3.6 & 0.26 & 8.7e-10 \\
\multirow{1}{16em}{100kb $(10.1\%)$} & 0.21 & 0.029 & 1.3e-13 & 2.4 & 0.095 & 2.2e-10 \\
 \hline
\end{tabular}
\end{table}


\newpage
\begin{table}
\caption{\bf{S-LDSC results for SNP annotations corresponding to \Robospan{}, \pRobospan{}, \Corspan{} and SEG-Blood gene sets for 6 autoimmune traits. Standardized Effect sizes ($\tau^{\star}$) and Enrichment (E) of $8$ SNP annotations corresponding to 4 gene sets (\Robospan{}, \pRobospan{}, \Corspan{} and SEG-Blood\cite{Finucane2018}) and 2 SNP annotations corresponding to 5kb and 100kb window based SNP-to-gene linking strategies for each gene set. Results for all annotations are conditional on $86$ baselineLD-v2.1 annotations. Reports are meta-analyzed across 6 Autoimmune traits.}}
\label{tab:Robocov_marginal_immune}  
\begin{tabular}[c]{|p{1.5cm}|p{0.5cm}|p{0.8cm}|p{0.8cm}|p{0.5cm}|p{0.8cm}|p{0.8cm}|} 
\hline
 \multicolumn{7}{c}{\Robospan{}}  \\ \hline
& $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.6\%)$} & 0.12 & 0.036 & 7e-04 & 2.7 & 0.25 & 5e-05 \\
\multirow{1}{16em}{100kb $(10\%)$} & 0.14 & 0.049 & 0.0051 & 2.3 & 0.19 & 2e-06 \\
\hline 
 \multicolumn{7}{c}{\pRobospan{}}  \\ \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.3\%)$} & 0.1 & 0.043 & 0.016 & 3.3 & 0.39 & 1e-04 \\
\multirow{1}{16em}{100kb $(9.9\%)$} & 0.11 & 0.059 & 0.061 & 2.5 & 0.24 & 1e-05 \\
\hline 
\multicolumn{7}{c}{\Corspan{}}  \\ \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.5\%)$} & 0.08 & 0.035 & 0.021 & 2.5 & 0.23 & 1e-04 \\
\multirow{1}{16em}{100kb $(9.8\%)$} & 0.037 & 0.035 & 0.28 &   2 & 0.15 & 8e-06 \\
 \hline 
\multicolumn{7}{c}{SEG-Blood}  \\ \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{5kb $(2.7\%)$} & 0.33 & 0.042 & 5e-15 & 4.2 & 0.25 & 8e-06 \\
\multirow{1}{16em}{100kb $(10.1\%)$} & 0.3 & 0.036 & 9e-17 & 2.7 & 0.11 & 7e-06 \\
 \hline
\end{tabular}
\end{table}

\newpage
\begin{table}[h]
\caption{\bf{Joint S-LDSC results for annotations corresponding to \Robospan{}, \pRobospan{}, \Corspan{} and SEG-Blood gene sets.}: Standardized Effect sizes ($\tau^{\star}$) and Enrichment (E) of  SNP annotations that are significant when all annotations from Table \ref{tab:Robocov_marginal} are modeled jointly and subjected to forward stepwise elimination. 2 annotations survive in the resulting joint model. The analysis is conditional on $86$ baselineLD-v2.1 annotations. Reports are meta-analyzed across $11$ Blood and Autoimmune traits.}
\label{tab:Robocov_joint} 
\begin{tabular}[c]{|p{2.1cm}|p{0.6cm}|p{0.7cm}|p{0.8cm}|p{0.7cm}|p{0.7cm}|p{0.8cm}|p{0.8cm}|} \hline
 & $\tau^{\star}$ & se($\tau^{\star}$) & p($\tau^{\star}$) & $E$ & se($E$) & p($E$) \\
\hline
\multirow{1}{16em}{Robospan(100kb)} & 0.1 & 0.03 & 1e-04 & 2.3 & 0.1 & 2e-09 \\
\multirow{1}{16em}{SEG-Blood(100kb)} & 0.2 & 0.03 & 2e-13 & 2.4 & 0.09 & 2e-10 \\
\hline 
\end{tabular}
\end{table}